common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -93,17 +93,48 @@
     <%doc>
         Loads a studio-frontend page, with the necessary context. Context is expected
         as a dictionary in the body of this tag.
-
-        Dev note: we could also add the locale-injection script in this block
-            -use a better default than the hardcoded 'en'. There should be a setting or something?
-            -lookup (webpack exported) locale-injection script using lang as key
-            -include it as the first script in this block
     </%doc>
     <%
         body = capture(caller.body)
+        body_dict = json.loads(body)
+        locale = body_dict['lang']
+        messages = "{}"
+        localeData = ""
```

Comment:
Nit: I'm not sure what the casing should be here - snake case or camel case - since it's HTML, but it looks like there is snake case and camel case here. I wonder if we could choose one and stick to it.

Reply:
This is a totally unintended view into the psyche of me as a python/js coder - `localeData` started out being defined inside of `<script type="application/json" id="SFE_i18n_localeScript">` (so naturally, a js variable is camelCased). `body_dict` only ever existed in a python context on these two lines (and naturally, a python variable is snake_cased). Wild. Keep those languages separated, kids.

I'll standardize as part of the refactor @thallada suggested.